package.json

```diff
@@ -63,34 +62,32 @@
     "uuid": "^3.3.2"
   },
   "scripts": {
-    "manualBuild": "node verifyCacheVersion.js && yarn localize && d2-manifest package.json ./public/manifest.webapp && REACT_APP_DHIS2_BASE_URL='../../..' react-scripts build && mkdirp buildzip/ && pushd build && bestzip ../buildzip/NewCaptureApp.zip * && popd",
-    "start": "react-scripts start",
-    "build": "node verifyCacheVersion.js && yarn localize && d2-manifest package.json ./public/manifest.webapp && GENERATE_SOURCEMAP=false react-scripts build && cp ./package.json ./build/package.json",
-    "test": "react-scripts test",
+    "start": "yarn verifyCacheVersion && d2-app-scripts start",
+    "start:forCypress": "node scripts/startAppForCypress.js",
+    "build": "yarn verifyCacheVersion && GENERATE_SOURCEMAP=false d2-app-scripts build && cp ./package.json ./build/app/package.json",
+    "build:manual": "yarn verifyCacheVersion && REACT_APP_DHIS2_BASE_URL='../../..' d2-app-scripts build",
```

Comment:
Why is this script necessary?

Reply:
It was used earlier to generate the prod app manually and upload it to an instance for testing. The app would be located in the /api/ endpoint so had to change `REACT_APP_DHIS2_BASE_URL`. Looks like this is not possible at the moment. Any way to achieve this? (without editing the output manually)

Reply:
@JoakimSM yeah, that's what `coreApp: false` would do.  Also, however, as soon as [this backend PR](https://github.com/dhis2/dhis2-core/pull/5977/files) gets in you'll be able to upload core apps to an instance and have them served at the core app URL (`dhis-web-capture`) so no need to build the app differently